lib/lambda_ethereum_consensus/fork_choice/helpers.ex

```diff
@@ -8,6 +8,30 @@ defmodule LambdaEthereumConsensus.ForkChoice.Helpers do
   alias SszTypes.Checkpoint
   alias SszTypes.Store
 
+  @spec current_status_message() ::
+          {:ok, SszTypes.StatusMessage.t()} | {:error, any}
+  def current_status_message do
```

Comment:
Since the function executed from the handler it is not possible to get the store as a param unfortunately

Reply:
You are right. Since we are doing some kind of computation inside, including it inside the Genserver may not be very convenient, what do you think of this @Arkenan?